client/job.lua

```diff
@@ -466,31 +469,33 @@ else
         end
 
         for i = 1, #sharedConfig.locations.armory do
-            local function enteredArmoryZone()
-                if QBX.PlayerData.job.onduty then
-                    lib.showTextUI(locale('text.armory_button'))
+            for ii = 1, #sharedConfig.locations.armory[i].locations do
```

Comment:
Why are we passing this through two loops?

Reply:
- armory is a table and you can create multiple armory / shop

https://github.com/Qbox-project/qbx_ambulancejob/blob/b27177372af7a24f3cf870c31d9cb6243ecfb7c9/config/shared.lua#L18-L35

- each location in the armory is a table too and you can add multiple locations for each armory

https://github.com/Qbox-project/qbx_ambulancejob/blob/b27177372af7a24f3cf870c31d9cb6243ecfb7c9/config/shared.lua#L31-L33